### PR TITLE
docs: fix README.md for slim action

### PR DESCRIPTION
**Pull request by chrisjohnson00**
## Proposed Changes

Fix to the README for how to run this action using the `slim` configuration.

## Readiness Checklist

In order to have this pull request merged, complete the following tasks.

### Pull request author tasks

- [X] I included all the needed documentation for this change.
- [X] I provided the necessary tests.
- [X] I squashed all the commits into a single commit.
- [X] I followed the [Conventional Commit v1.0.0 spec](https://www.conventionalcommits.org/en/v1.0.0/).
- [X] If this is a breaking change, write upgrade instructions in the [upgrade guide](../docs/upgrade-guide.md).

### Super-linter maintainer tasks

- [ ] Label as `breaking` if this change breaks compatibility with the previous released version.
- [ ] Label as either: `automation`, `bug`, `documentation`, `enhancement`, `infrastructure`.

### CODE REVIEW

**Comment by chrisjohnson00** [comment]
Here is where I'm using it 

https://github.com/chrisjohnson00/plex-sad-search/blob/16b70b09cb4396bd6437c550b1d33b0a682ae96b/.github/workflows/ci.yml#L151

AFAIK, the standard usage reference is correct. However, I don't use it since the container pull takes so dang long. 

I will check (on phone now) later.

File: README.md [hunk 1]
```diff
@@ -142,7 +142,7 @@ For more information, see
 Super-Linter provides several variants:
 
 - `standard`: `super-linter/super-linter:[VERSION]`: includes all supported linters.
```
Comment (ferrarimarco) [inline README.md:144]: Can you please refactor this one to: `super-linter/super-linter@[VERSION]`?

Thanks!

**Review by ferrarimarco — CHANGES_REQUESTED** [review]
One small change and we should be good to go.

File: README.md (hunk 1 above)
Comment (chrisjohnson00) [reply README.md:144]: oh yeah, good catch

**Review by ferrarimarco — APPROVED** [review]

**Comment by ferrarimarco** [comment]
@chrisjohnson00 last thing to do: I've rebased this on the latest `main`, but we also need this to be a single commit. Can you please squash the two commits in a single one? Thanks a lot!

**Comment by chrisjohnson00** [comment]
> @chrisjohnson00 last thing to do: I've rebased this on the latest `main`, but we also need this to be a single commit. Can you please squash the two commits in a single one? Thanks a lot!

Done!

**Pull request merged**